odoo/cli/i18n.py

```diff
@@ -0,0 +1,228 @@
+import argparse
+import csv
+import logging
+import os
+import sys
+from pathlib import Path
+
+import odoo
+from odoo.cli.command import Command
+from odoo.modules import get_module_path
+from odoo.modules.registry import Registry
+from odoo.tools import config
+from odoo.tools.translate import load_language, trans_export
+
+_logger = logging.getLogger(__name__)
+
+EXPORT_EXTENSIONS = ['.po', '.pot', '.tgz', '.csv']
+IMPORT_EXTENSIONS = ['.po', '.csv']
+
+
+class SubcommandHelpFormatter(argparse.RawTextHelpFormatter):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs, max_help_position=80)
+
+
+class I18n(Command):
+    """ Import, export, setup languages and internationalization files """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        subparsers = self.parser.add_subparsers(
+            dest='subcommand', required=True,
+            help='Subcommands help')
+
+        self.import_parser = subparsers.add_parser(
+            'import',
+            help="Import i18n files",
+            description="Imports provided translation files",
+            formatter_class=SubcommandHelpFormatter,
+        )
+        self.export_parser = subparsers.add_parser(
+            'export',
+            help="Export i18n files",
+            description="Exports language files into the i18n folder of each module",
+            formatter_class=SubcommandHelpFormatter,
+        )
+        self.loadlang_parser = subparsers.add_parser(
+            'loadlang',
+            help="Load languages",
+            description="Loads languages",
+            formatter_class=SubcommandHelpFormatter,
+        )
+
+        for parser in (self.import_parser, self.export_parser, self.loadlang_parser):
+            parser.add_argument(
+                '-c', '--config', dest='config',
+                help="use a specific configuration file")
+            parser.add_argument(
+                '-d', '--database', dest='db_name', default=None,
+                help="database name, connection details will be taken from the config file")
```

Comment:
5710ba712c3c50a74a816f1270d159b62f3a1b95

we should parse the option as a CSV and assert there's 0 or 1 db in the resulting list, otherwise one could type `-d one,two,three`